cmd/anonymizer/app/anonymizer/anonymizer.go

```diff
@@ -53,26 +53,32 @@ type mapping struct {
 //
 // The mapping from original to obfuscated strings is stored in a file and can be reused between runs.
 type Anonymizer struct {
-	mappingFile      string
-	logger           *zap.Logger
-	lock             sync.Mutex
-	mapping          mapping
-	hashStandardTags bool
-	hashCustomTags   bool
-	hashLogs         bool
-	hashProcess      bool
+	mappingFile string
+	logger      *zap.Logger
+	lock        sync.Mutex
+	mapping     mapping
+	options     Options
+}
+
+// Options represents the various options with which the anonymizer can be configured.
+type Options struct {
```

Comment:
Do you need to have yaml tags on the fields in this struct, similar to writer.Config?